autosklearn/evaluation/__init__.py

```diff
@@ -261,8 +266,15 @@ def run(
         if self.init_params is not None:
             init_params.update(self.init_params)
 
+        if self.port is None:
```

Comment:
The handling of `port is None` is different for this logger and the logger above. Can this be unified?

Reply:
The only reason this is not unified is to get a different name for the TAE and pynisher call. Is it ok if we only get TAE in the message name?

Reply:
Sorry, I meant that in one case it check if the port is None, in the other it doesn't check.